lxd/network/acl/driver_common.go

```diff
@@ -751,33 +751,46 @@ func (d *common) Delete() error {
 // GetLog gets the ACL log.
 func (d *common) GetLog(clientType request.ClientType) (string, error) {
 	// ACLs aren't specific to a particular network type but the log only works with OVN.
-	logPath := shared.HostPath("/var/log/ovn/ovn-controller.log")
-	if !shared.PathExists(logPath) {
-		return "", fmt.Errorf("Only OVN log entries may be retrieved at this time")
-	}
-
-	// Open the log file.
-	logFile, err := os.Open(logPath)
+	targetPath, err := os.Readlink("/run/openvswitch")
```

Comment:
what if this path doesn't exist, this would break existing installations that do have `/var/log/ovn/ovn-controller.log"` right?

Reply:
hmmm true. I initially thought that this symlink would be created even with OVN built in, but this isn't the case.. l'll fix that.

Reply:
we do have to consider non-snap installs too